src/WooCommerce.php

```diff
@@ -1159,6 +1159,7 @@ public static function woocommerce_checkout_order_processed($order_id, $posted_d
 
       if ($subscriber = WC()->session->get('subscriber_data')) {
         $purchase['subscriber_id'] = $subscriber['subscriber_id'];
+        Server::checkPurchase($purchase['id'], $purchase['subscriber_id']);
```

Comment:
Should be limited to users with `alfa_dummy_address` only

Reply:
Resolved in https://github.com/netzstrategen/wordpress-ssofact/pull/42/commits/fbafceeece8fa4afa48007b5f135efbd4b358acc